api/app/serializers/v1/export_target_serializer.rb

```diff
@@ -2,16 +2,45 @@ module V1
   class ExportTargetSerializer < ManifoldSerializer
     include ::V1::Concerns::ManifoldSerializer
 
-    typed_attribute :strategy, Types::String
+    CONFIGURATION_HASH = Types::Hash.schema(
+      sftpKey: Types::Hash.schema(
+        port: Types::Serializer::Port,
+        host: Types::Serializer::Host,
+        username: Types::String,
+        compression: Types::Integer,
+        privateKey: Types::String.meta(
+          description: "The key used to secure the sftp connection. This value "\
+          "can be updated, but will not be sent in any response. It will instead "\
+          "respond with [FILTERED] if there is a value.",
+          example: "[FILTERED]"
+        )
+      ).optional,
+      sftpPassword: Types::Hash.schema(
+        port: Types::Serializer::Port,
+        host: Types::Serializer::Host,
+        username: Types::String,
+        password: Types::String
+      ).optional,
+      targetNameFormat: Types::String.meta(example: "%s-%t.%e"),
+      strategy: Types::Integer.meta(
+        description: "A digit representation of the export target strategy",
+        read_only: true
+      )
+    )
+
+    typed_attribute :strategy, Types::String.meta(
+      description: "Denotes which on the two sftp configurations will be used.",
```

Comment:
Please correct typo (of, not on). Remove period because this is not a complete sentence.